### ISSUE TITLE
duckstation emu app stop working

### ISSUE THREAD

**Issue by shikulja** [leecher1337/ntvdmx64#112]
after install NTVDMx64 (Leecher1337 build) stop working https://github.com/stenzek/duckstation
help uninstall NTVDMx64

the error is related to hijacking applications and there may be problems with other applications.

**Comment by leecher1337** [comment]
Application can be ininstalled like every other application via control panel -> Add/Remove programs

**Comment by shikulja** [comment]
yes I know. but I'm just saying that it's a good idea to fix it if this error repeats for everyone

**Comment by leecher1337** [comment]
Your error report doesn't contain any usable details. 
You need to provide at least a memory dump and stack trace of crashes (including debug symbols, otherwise they are worthless) and step-by-step intructions on how to reproduce the error (which you didn't even describe).

If application doesn't allow process injection, it may be unfixable. In this case, the only thing that may be done is maintaining a list in the registry about executables where injection shouldn't occur and make the loader support an exclusion list.

**Comment by leecher1337** [comment]
For your reference: NTVDMx64 (Running Jewel Thief on WOW32) and Duckstation running happily together on Win8:

![image](https://user-images.githubusercontent.com/6495842/103808302-11265100-5058-11eb-8488-2a7c1ea4729c.png)

**Comment by shikulja** [comment]
@leecher1337  
thanks for checking.
looks like i was having problems with registry and paths

**Comment by shikulja** [comment]
https://github.com/stenzek/duckstation/pull/1394

**Comment by leecher1337** [comment]
What does this have to do with NTVDMx64?  

NTVDMx64 needs ConhostV1 and ConhostV2 or this weird new console from M$ are unlikely to be supported, because they don't have a console Framebuffer enabled and therefore they won't work with graphics anymore. See my related proposal to M$: https://github.com/microsoft/terminal/issues/246
